src/main/scala/org/hammerlab/guacamole/commands/StructuralVariantCaller.scala

```diff
@@ -180,6 +182,99 @@ object StructuralVariant {
       graph
     }
 
+    case class SVClique(
+      reads: Set[PairedMappedRead],
+      span: GenomeRange,
+      wiggle: Long
+    )
+
+    def findOneClique(g: PairedReadGraph, maxNormalInsertSize: Int): SVClique = {
+      // Seed the clique with the source of the lowest-weight edge.
+      val edges = g.edges.toList.sortBy(_.weight)
+      val bestEdge = edges.head
+
+      val initPair = bestEdge.nodes.minBy(_.minPos).value
+      var (pairMin, svStart, svEnd, _) = initPair.startsAndStops
+
+      // this is the "wiggle room": by how much can the SV shrink and still be compatible with all reads in the clique?
+      var wiggle = initPair.insertSize - maxNormalInsertSize - (svEnd - svStart)
```

Comment:
This and later calculations could possibly be clarified by giving `PairedMappedRead` a `.readsLength` method equal to `insertSize - gapSize`.